starwars/src/Components/CharCard/CharCard.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import './CharCard.css';
+import { Card, CardImg, CardText, CardBody, CardTitle, CardSubtitle, Button, UncontrolledCarousel } from "reactstrap";
```

Comment:
You've got some unused variables here `CardImg`, `CardBody`, `CardSubtitle`, etc.